src/components/JumpIn/SecondaryPin.tsx

```diff
@@ -1,14 +0,0 @@
-import React from "react"
```

Comment:
By removing these components, we're introducing a breaking change, do you know if any of these icons are used elsewhere?

Reply:
not used anywere